### DISCUSSION TITLE
How do I set the time zone to match my sql server database time zone?

### DISCUSSION THREAD

**Discussion by nicholasvperry** [prisma/prisma#18405] [Q&A] [answered]
I am trying to set up Prisma to use my current database time zone which is eastern time zone. I can see that prisma is pulling in the correct time, but it is setting it to Z instead of my local time zone which is 4 hours different. I apologize if this is already answered or in the docs, I have looked for many hours before posting and I cannot find a solution. Below one of my models. Both @db.DateTime and @db.Date are returning dates with a 4 hour difference. Also when I save a report it adds 4 hours to the actual time. I know if I can set the time zone in my prisma client it will solve the issue. Also I cannot change the timezone on my db so doing it through prisma is my only option.

```prisma
model TurnReport {
  entry_id   Int      @id(map: "PK_TurnReport") @default(autoincrement())
  comment    String   @db.Text
  datetime   DateTime @db.DateTime
  author     String?  @db.VarChar(100)
  area_id    Int
  shift_date DateTime @db.Date
  shift_num  Int      @db.TinyInt
  ip         String?  @db.VarChar(32)
  TurnReport_Attachments TurnReport_Attachments[] 
}
```

**Comment by ludralph** [comment]
Hi @nicholasvperry 👋 

You would need to convert the timestamp to a specific required timezone as Prisma would save it in UTC in the DB. I saw an open [issue](https://github.com/prisma/prisma/issues/5051) for improving timezone support for MySQL but couldn't find one for SQLServer. I also noticed a work around in that issue which is documented [here](https://github.com/prisma/prisma/issues/5051#issuecomment-1279790199). You might want to take a look and see if the workaround applies to SqlServer.

**Reply by nicholasvperry** [reply]
Hi @ludralph 👋
Thanks for the reply. I did see that and I tried and unfortunately it did not work with sqlserver.

**Reply by ludralph** [reply, marked as answer]
Hi @nicholasvperry, Can you please take a look at the workaround in this [issue comment](https://github.com/prisma/prisma/issues/5051#issuecomment-1468168953)

**Reply by ludralph** [reply]
Hi @nicholasvperry 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
